src/networking.c

```diff
@@ -1691,10 +1691,82 @@ client *lookupClientByID(uint64_t id) {
     return (c == raxNotFound) ? NULL : c;
 }
 
+/* This function should be called from _writeToClient when the reply list is not empty,
+ * it gathers the scattered buffers from reply list and sends them away with connWritev.
+ * If we write successfully, it returns C_OK, otherwise, C_ERR is returned,
+ * and 'nwritten' is an output parameter, it means how many bytes server write
+ * to client. */
+int _writevToClient(client *c, ssize_t *nwritten) {
```

Comment:
Please mark function as static (... I know that several functions in this file could have use `static`, and yet.)

Reply:
Done.